bosco_cluster

```diff
@@ -932,7 +933,7 @@ fi
 #TODO: do we need a separate sandbox for each cluster?
 remote_base_dir_root="bosco/cluster"
 remote_base_dir_default="$remote_base_dir_root/default"
-remote_base_dir_host="bosco"
+[[ -n "$remote_base_dir_host" ]] || remote_base_dir_host="bosco"
```

Comment:
```suggestion
[[ $remote_base_dir_host ]] || remote_base_dir_host="bosco"
```

It'll work fine how you have it, but the beauty of the double brackets is that its safe to omit the quotes when testing a string.

Also the quotes around the string "bosco" are superfluous in this and the above lines.

On the other hand, quotes are _missing_ below in the mkdir command that actually uses `$remote_base_dir_host` ... (that is, where it actually does matter)